pkg/ui/terminal.go

```diff
@@ -420,3 +425,44 @@ func (u *TerminalUI) handleMessage(msg *api.Message) {
 func (u *TerminalUI) ClearScreen() {
 	fmt.Print("\033[H\033[2J")
 }
+
+func formatToolCallResponse(payload any) string {
```

Comment:
nit: let it always return the non-truncated string, and caller can call truncateString on the ouput.